glean/metrics.yaml

```diff
@@ -332,6 +332,34 @@ sample:
           `authenticationinapp.h`.
         type: string
 
+  reauthentication_inapp_step:
```

Comment:
Can we have 1 for account-deletion and 1 for subscription? Talk with Sarah to see what is better.

Reply:
We landed on separate events for each authentication flow.